test/sharness/t0116-prometheus-data/prometheus_metrics

```diff
@@ -159,88 +159,19 @@ flatfs_datastore_sync_latency_seconds_bucket
 flatfs_datastore_sync_latency_seconds_count
 flatfs_datastore_sync_latency_seconds_sum
 flatfs_datastore_sync_total
-go_cgo_go_to_c_calls_calls_total
-go_gc_cycles_automatic_gc_cycles_total
-go_gc_cycles_forced_gc_cycles_total
-go_gc_cycles_total_gc_cycles_total
```

Comment:
We can't configure thoses sadly because they are already registered at init by the lib:
https://github.com/prometheus/client_golang/blob/9801a4e3ceb49d24dfaf4fa6fc8c2d58aa3b2dc9/prometheus/registry.go#L61-L64
(so if we try to do so it just panic)

Reply:
@Jorropo  oof, is this upstream issue?

We are handling this gracefully for our own metrics: https://github.com/ipfs/kubo/blob/v0.16.0/core/corehttp/gateway_handler.go#L181-L187